context/app/static/js/shared-styles/Links/iconLinks/EmailIconLink/EmailIconLink.jsx

```diff
@@ -1,15 +1,10 @@
 import React from 'react';
 import IconLink from 'js/shared-styles/Links/iconLinks/IconLink';
 import { EmailIcon } from 'js/shared-styles/icons';
-import { trackEvent } from 'js/helpers/trackers';
+import { trackLink } from 'js/helpers/trackers';
 
 function sendEmailEvent(event) {
-  trackEvent({
-    category: 'Email Link',
-    action: 'Clicked',
-    label: event.target.href,
-    nonInteraction: false,
-  });
+  trackLink(event.target.href);
```

Comment:
Seems like we're losing value here and no longer categorize our link events?

Reply:
The Matomo API sees to have a few more functions, for different types of events, but those functions are also less flexible. Looking at the [docs](https://developer.matomo.org/api-reference/tracking-javascript):
> `trackLink( url, linkType )` - Log a click from your own code. url is the full URL which is to be tracked as a click. linkType can either be 'link' for an outlink or 'download' for a download.

In this particular case, maybe the matomo way would be to filter based on the URL?

Reply:
Should we provide a `linkType` arg?

Reply:
I'd want to make the `linkType` "email", but it says that's not allowed? But what happens if we do? I'd like to merge this as-is, but I'll file a follow-up issue.